src/AWSS3.jl

```diff
@@ -355,16 +364,16 @@ s3_copy(a...; b...) = s3_copy(global_aws_config(), a...; b...)
 [PUT Bucket](http://docs.aws.amazon.com/AmazonS3/latest/API/RESTBucketPUT.html)
 """
 function s3_create_bucket(aws::AbstractAWSConfig, bucket; kwargs...)
-    @protected try
+    r = @protected try
         if aws.region == "us-east-1"
             S3.create_bucket(bucket; aws_config=aws, kwargs...)
         else
             bucket_config = """
-                                <CreateBucketConfiguration
-                                            xmlns="http://s3.amazonaws.com/doc/2006-03-01/">
-                                    <LocationConstraint>$(aws.region)</LocationConstraint>
-                                </CreateBucketConfiguration>
-                            """
+            <CreateBucketConfiguration
+            xmlns="http://s3.amazonaws.com/doc/2006-03-01/">
+            <LocationConstraint>$(aws.region)</LocationConstraint>
+            </CreateBucketConfiguration>
+            """
```

Comment:
Should be within then 92-character line limit
```suggestion
                <CreateBucketConfiguration xmlns="http://s3.amazonaws.com/doc/2006-03-01/">
                    <LocationConstraint>$(aws.region)</LocationConstraint>
                </CreateBucketConfiguration>
                """
```